### DISCUSSION TITLE
How to activate email notifications for my topics in the forum?

### DISCUSSION THREAD

**Discussion by vitalykarasik** [community/community#23443] [New to GitHub]
<p>I’d like to receive email notifications when someone replies to my topics. I went over my ‘notifications’ and ‘preferences’ but couldn’t find relevant checkbox.<br>
Please suggest,<br>
Vitaly</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>when you create a new topic, it says,</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077113-3c63a9c9-9bbe-42e4-81e2-2684c0ac996d.png" alt="image" data-base62-sha1="nQkYXBaH1fgxPEgIzhYsEh89DvM" width="554" height="57"></p>
<p>then at <code>email</code> preferences, select <code>always</code></p>
<p><a href="https://github.community/u/jdevstatic/preferences/emails">https://github.community/u/jdevstatic/preferences/emails</a></p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077114-ef3b10e2-769e-4799-b516-fd460bfce325.png" alt="image" data-base62-sha1="nD7eyGWrdZlp7lnVSo3NORSfi4I" width="510" height="205"></p>
<p>don’t forget to save the changes</p>

**Comment by vitalykarasik** [comment]
<p>Many thanks - somehow I missed this ‘emails’ menu.</p>
